nucliadb_vectors/src/data_point_provider/merge_worker.rs

```diff
@@ -103,7 +100,7 @@ impl Worker {
         let creates_work = state.replace_work_unit(new_dp);
         fs_state::persist_state(&lock, &state)?;
         std::mem::drop(lock);
-        info!("Merge on {subscriber:?}:\n{report}");
+        tracing::info!("Merge on {subscriber:?}:\n{report}");
```

Comment:
Why are you adding this? If logging it's not imported, there's nothing confusing right?

Reply:
I forgot it was imported. Fixed at [d846d76](https://github.com/nuclia/nucliadb/pull/600/commits/d846d76971f64cd8d803c4562c05718265a55ea3)